src/server/wesnothd/game.cpp

```diff
@@ -1955,6 +1955,12 @@ void game::send_server_message(const char* message, std::optional<player_iterato
 
 		msg.set_attr("id", "server");
 		msg.set_attr_dup("message", message);
+		const std::time_t time = ::std::time(nullptr);
+		std::stringstream ss;
+		ss << time;
+		const std::string timestr = ss.str();
+		const char* timecstr = timestr.c_str();
+		msg.set_attr_dup("time", timecstr);
```

Comment:
I think it's ok if we write this a little shorter:
```suggestion
		std::stringstream ss;
		ss << ::std::time(nullptr);
		msg.set_attr_dup("time", ss.str().c_str());
```

Reply:
I'm unsure where the string object returned by ss.str() in the middle of the expression will be destroyed. If it's destroyed before set_attr_dup copying the char array, the array will be already freed memory. But if my worry is unfounded, I'll shorten it like you suggest ofc

Reply:
If it would get destroyed in the middle of the expression that'd be pretty crazy. You can find an example of similar code in this same source file, btw.

https://en.cppreference.com/w/cpp/language/lifetime#Temporary_object_lifetime

Reply:
Alright, I've applied the change